pkg/operator/api/userconfig/apis.go

```diff
@@ -40,13 +42,16 @@ var apiValidation = &cr.StructValidation{
 			},
 		},
 		{
-			StructField:  "ModelName",
-			DefaultField: "Name",
+			StructField: "ModelName",
 			StringValidation: &cr.StringValidation{
-				Required:                   false,
-				AlphaNumericDashUnderscore: true,
+				AllowEmpty:                           true,
+				AlphaNumericDashDotUnderscoreOrEmpty: true,
 			},
 		},
+		{
+			StructField:         "ModelPath",
+			StringPtrValidation: &configreader.StringPtrValidation{},
+		},
```

Comment:
Can you add validation that both of these fields aren't specified? You can use `ErrorSpecifyOnlyOne()`, and the structure will look like it does for e.g. models:

```go
func (models Models) Validate() error {
	for _, model := range models {
		if err := model.Validate(); err != nil {
			return err
		}
	}

	resources := make([]Resource, len(models))
	for i, res := range models {
		resources[i] = res
	}

	dups := FindDuplicateResourceName(resources...)
	if len(dups) > 0 {
		return ErrorDuplicateResourceName(dups...)
	}

	return nil
}

func (model *Model) Validate() error {
	...
}
```